### ISSUE TITLE
Cannot parse the file list. It seems like Steam has update the webpage. Please update to the latest version or notify the author

### ISSUE THREAD

**Issue by shellshock1953** [pyscsd/steamCloudSaveDownloader#6]
**Describe the bug**
Most likely its Witcher 3 (based on context, because it is failing right after Wicther 1, next Witcher 2 and crash), but I am not 100% sure.

**To Reproduce**
Using docker-compose to run script:
next compose file is used:   
```yaml
services:
  scsd:
    image: hhhhojeihsu/scsd:latest
    container_name: scsd
    volumes:
      - ./steam_save:/data
      - ./scsd/:/config
      - /etc/localtime:/etc/localtime:ro
```

```sh
docker-compose -f scsd.yaml up -d --force-recreate
```

**Screenshots**
Witcher 3
![image](https://github.com/user-attachments/assets/fdf6df3d-5534-4d39-954b-7cffdbe94dd9)

**Environment:**
config:
```ini
[General]
save_dir = /data

[Log]
log_level = 2

[Rotation]
rotation = 15

[Notifier]
notifier = script
path = /config/notify.sh
notify_if_no_change = false
level = 1

[Target]
mode = exclude
list = 7, 12
```

/config/notify.sh
```sh
#!/bin/sh
wget -q -O - --post-data='Steam Backups - done '"${1}"'' ntfy.sh/REDACTED > /dev/null
```

content of steam save dir:
```
ls steam_save
drwxr-xr-x    - bsukho 13 Aug 12:39 70
drwxr-xr-x    - bsukho 13 Aug 12:40 130
drwxr-xr-x    - bsukho 13 Aug 12:40 240
drwxr-xr-x    - bsukho 13 Aug 12:42 280
drwxr-xr-x    - bsukho 13 Aug 12:42 400
drwxr-xr-x    - bsukho 13 Aug 12:42 440
drwxr-xr-x    - bsukho 13 Aug 12:42 550
drwxr-xr-x    - bsukho 13 Aug 12:42 570
drwxr-xr-x    - bsukho 13 Aug 12:43 620
drwxr-xr-x    - bsukho 13 Aug 12:43 730
drwxr-xr-x    - bsukho 13 Aug 12:43 2320
drwxr-xr-x    - bsukho 13 Aug 12:43 20900
.rw-r--r--  77k bsukho 13 Aug 15:55 scsd.log
.rw-r--r--  41k bsukho 13 Aug 15:55 scsd.sqlite3
.rw------- 8.6k bsukho 13 Aug 15:54 session.sb
```

**Log file and config :**

<details>
  <summary>Log:2</summary>
  
```txt
[run.sh] Config specified /config/scsd.conf
2024-08-13 15:51:35 [INFO] Config read from '/config/scsd.conf'
2024-08-13 15:51:35 [INFO] Options: {'General': {'save_dir': '/data', '2fa': 'mobile'}, 'Rotation': {'rotation': 15}, 'Log': {'log_level': 2}, 'Notifier': {'notify_if_no_change': True, 'notifier': 'script', 'webhook': '', 'path': '/config/notify.sh', 'level': 1}, 'Target': {'mode': 'exclude', 'list': [7, 12]}, 'Danger Zone': {'wait_interval': [3, 5]}}
2024-08-13 15:51:35 [INFO] CWD: /
2024-08-13 15:51:35 [INFO] Files will be saved to '/data'
2024-08-13 15:51:35 [INFO] Refreshing session
2024-08-13 15:51:43 [INFO] Getting Game Save List
2024-08-13 15:51:43 [INFO] Random sleep for 5 seconds
2024-08-13 15:51:51 [INFO] (1/126) Processing Half-Life
2024-08-13 15:51:51 [INFO] Random sleep for 3 seconds
2024-08-13 15:51:54 [INFO] (2/126) Processing Half-Life: Blue Shift
2024-08-13 15:51:54 [INFO] Random sleep for 3 seconds
2024-08-13 15:51:58 [INFO] (3/126) Processing Counter-Strike: Source
2024-08-13 15:51:58 [INFO] Random sleep for 3 seconds
2024-08-13 15:52:01 [INFO] (4/126) Processing Half-Life: Source
2024-08-13 15:52:01 [INFO] Random sleep for 3 seconds
2024-08-13 15:52:05 [INFO] (5/126) Processing Portal
2024-08-13 15:52:05 [INFO] Random sleep for 3 seconds
2024-08-13 15:52:08 [INFO] (6/126) Processing Team Fortress 2
2024-08-13 15:52:08 [INFO] Random sleep for 4 seconds
2024-08-13 15:52:12 [INFO] (7/126) Processing Left 4 Dead 2
2024-08-13 15:52:12 [INFO] Random sleep for 5 seconds
2024-08-13 15:52:18 [INFO] (8/126) Processing Dota 2
2024-08-13 15:52:18 [INFO] Random sleep for 3 seconds
2024-08-13 15:52:21 [INFO] (9/126) Processing Portal 2
2024-08-13 15:52:21 [INFO] Random sleep for 4 seconds
2024-08-13 15:52:26 [INFO] (10/126) Processing Counter-Strike 2
2024-08-13 15:52:26 [INFO] Random sleep for 3 seconds
2024-08-13 15:52:29 [INFO] (11/126) Processing Quake II
2024-08-13 15:52:29 [INFO] Random sleep for 5 seconds
2024-08-13 15:52:34 [INFO] (12/126) Processing The Witcher: Enhanced Edition
2024-08-13 15:52:34 [INFO] Random sleep for 4 seconds
2024-08-13 15:52:39 [INFO] (13/126) Processing The Witcher 2: Assassins of Kings Enhanced Edition
2024-08-13 15:52:39 [INFO] Random sleep for 3 seconds
2024-08-13 15:52:42 [INFO] Random sleep for 3 seconds
2024-08-13 15:52:46 [ERROR] Cannot parse the file list. It seems like Steam has update the webpage. Please update to the latest version or notify the author.
2024-08-13 15:52:46 [INFO] Retrying in 3 seconds
2024-08-13 15:52:46 [INFO] Random sleep for 3 seconds
2024-08-13 15:52:49 [ERROR] Cannot parse the file list. It seems like Steam has update the webpage. Please update to the latest version or notify the author.
2024-08-13 15:52:49 [INFO] Retrying in 4 seconds
2024-08-13 15:52:49 [INFO] Random sleep for 4 seconds
2024-08-13 15:52:54 [ERROR] Cannot parse the file list. It seems like Steam has update the webpage. Please update to the latest version or notify the author.
2024-08-13 15:52:54 [INFO] Retrying in 4 seconds
2024-08-13 15:52:54 [ERROR] Maximum attempt reached. Aborting
2024-08-13 15:52:55 [ERROR] Cannot parse the file list. It seems like Steam has update the webpage. Please update to the latest version or notify the author.
```
</details>

There is no need to attach Log:3 because there is no additional info related to the issue - just list of already existing save-files and nothing about which game crushes. But if needed - I can attach those

**Additional context**
Nothing

**Comment by hhhhhojeihsu** [comment]
Hi shellshock,

It seems like your directory does not contain the folder 20920, which is the appID for Witcher 2. So I suspect the error happens when parsing the webpage for Witcher 2.

Could you upload the html file for the link to the storage for [Witcher 2](https://store.steampowered.com/account/remotestorageapp/?appid=20920&l=english)? You can remove the private information (steamID, link to your save file etc.) if you want.

**Comment by shellshock1953** [comment]
Here you go: [20920.zip](https://github.com/user-attachments/files/16633768/20920.zip)
It consist of 2 pages

**Comment by hhhhhojeihsu** [comment]
I suspect the issue is caused by games that have more than 50 save files and in non-English environment. I forget to add the language specifier for link got from next page.

Should be solved in [v0.0.76](https://github.com/pyscsd/steamCloudSaveDownloader/releases/tag/v0.0.76). Please pull the latest docker image and try it again.
